.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+# Dependabot files PRs by pushing branches named
+# "dependabot/{ecosystem}/{depname}".  What it pushes to those
+# branches is pretty naive; it doesn't support us telling it that it
+# needs to run "make generate" after making an edit.
+#
+# But we can work around that by creating a workflow run "make
+# generate" on those branches!
+name: dependabot
+"on":
+  push:
+    branches:
+      - 'dependabot/**'
+
+jobs:
+  repatriate:
+    runs-on: ubuntu-latest
+    name: repatriate
```

Comment:
bike shed: not sure this represents what it is doing and adds confusion since we removed repatriation from the workflow. Maybe just bluntly `generate` or `make-generate`

Reply:
I've changed it to be named `generate`.